api/GlobalEventHandlers.json

```diff
@@ -3609,7 +3609,7 @@
               "version_added": "45"
             },
             "edge": {
-              "version_added": "12"
+              "version_added": "79"
```

Comment:
Can you double check why this was originally 12? Maybe the event itself is supported?

Reply:
The `12` came from mirroring from Internet Explorer, it looks like.  However, since support was disconfirmed in IE, this data is also inaccurate.

Reply:
I see, thanks!